packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -137,53 +55,27 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
   }
 
   private _navigationLinks(): JSX.Element {
-    const links: Array<JSX.Element> = [];
-    const { bestPractices, dos, donts } = this.props;
+    const props = this.props;
 
-    if (bestPractices && dos && donts) {
-      links.push(
-        <div className="ComponentPage-navLink" key="nav-link">
-          <Link href={this._baseUrl + '#BestPractices'}>Best Practices</Link>
-        </div>
-      );
-    }
+    const sections = [
```

Comment:
This gives (nearly) the exact same result as the previous code in a much more compact way. Only difference is that we now display the "Best Practices" link if `bestPractices || (dos && donts)` rather than if `bestPractices && dos && donts`--could revert this part if desired. (A literal "Best Practices" heading is only shown later if `bestPractices` is provided.)

Reply:
This is fine I think...